sycl/test-e2e/bindless_images/sampled_fetch/fetch_2D_USM.cpp

```diff
@@ -1,8 +1,7 @@
-// REQUIRES: cuda
-// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_2d_usm
+// REQUIRES: aspect-ext_oneapi_bindless_images
 
```

Comment:
Same as above, why is `// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_2d_usm` being removed?

Reply:
Aspect has been readded and enabled in the L0 backend. Resolving.